### PR TITLE
Fix failing tests on master - adding more time to timeout

### PR DESCRIPTION
**Pull request by idanizi-panw**
at ci:
image

locally
image

because of unstable test

### CODE REVIEW

File: tests/containers/ReportContainer_spec.js [hunk 1]
```diff
@@ -310,7 +310,7 @@ describe('Report Container', () => {
     const toRender = <ReportContainer sections={prepareSections(testTemplate)} />;
     const reportContainer = mount(toRender);
 
-    await new Promise(resolve => setTimeout(resolve, 5001));
```
Comment (asafshen) [inline tests/containers/ReportContainer_spec.js:313]: imo 10000 is more readable for humans

**Comment by dorsha** [comment]
I don't understand the fix, please explain better. And why you choose this value for the timeout?

**Comment by dorsha** [comment]
Also, what has changed that needed this change?

File: tests/containers/ReportContainer_spec.js (hunk 1 above)
Comment (idanizi-panw) [reply tests/containers/ReportContainer_spec.js:313]: will change

**Comment by idanizi-panw** [comment]
> I don't understand the fix, please explain better. And why you choose this value for the timeout?

this test wasn't stable, needed more time for the report to render before checking expected.

> Also, what has changed that needed this change?

we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow

**Comment by dorsha** [comment]
> > I don't understand the fix, please explain better. And why you choose this value for the timeout?
> 
> this test wasn't stable, needed more time for the report to render before checking expected.
> 
> > Also, what has changed that needed this change?
> 
> we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow

So the test was stable, what is the new time was added to the AutoSizer?

**Comment by idanizi-panw** [comment]
> > > I don't understand the fix, please explain better. And why you choose this value for the timeout?
> > 
> > 
> > this test wasn't stable, needed more time for the report to render before checking expected.
> > > Also, what has changed that needed this change?
> > 
> > 
> > we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow
> 
> So the test was stable, what is the new time was added to the AutoSizer?

added 500ms instead of 100ms:
https://github.com/demisto/sane-reports/pull/195/files#diff-0e672ae5527a671a5aa65b657426c852c801bdc126cfb15da4da31528135593aR14

**Review by asafshen — APPROVED** [review]

**Pull request merged**